redash/query_runner/presto.py

```diff
@@ -76,9 +78,18 @@ def run_query(self, query):
 
         cursor = connection.cursor()
 
+        # If a column appears more than once with the same name, subsequent
+        # instances are numbered 1, 2, , ... N.
+        columns_seen = defaultdict(int)
+        def make_unique(name):
+          columns_seen[name] += 1
+          if columns_seen[name] == 1:
+            return name
+          return name + str(columns_seen[name] - 1)
+
         try:
             cursor.execute(query)
-            columns_data = [(row[0], row[1]) for row in cursor.description]
+            columns_data = [(make_unique(row[0]), row[1]) for row in cursor.description]
```

Comment:
Thanks for pointing that method out, I've made the change.

I'm hesitant to change BaseQueryRunner's fetch_columns. Turns out my approach renumbers things differently than BaseQueryRunner (me: foo, foo1, bar, bar1, basequeryrunner: foo, foo1, bar, bar2). This renaming of columns would likely break visualizations that had stored the old name.